src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -35,25 +35,25 @@ public List<Manufacturer> getAll() {
     public Optional<Manufacturer> get(Long id) {
         String getRequest =
                 "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = false";
+        Manufacturer manufacturer = null;
         try (Connection connection = ConnectionUtil.getConnection();
-                 PreparedStatement getManufacturersStatement =
+                 PreparedStatement getManufacturerStatement =
                         connection.prepareStatement(getRequest)) {
-            getManufacturersStatement.setLong(1, id);
-            ResultSet resultSet = getManufacturersStatement.executeQuery();
-            Manufacturer manufacturer = null;
+            getManufacturerStatement.setLong(1, id);
+            ResultSet resultSet = getManufacturerStatement.executeQuery();
             if (resultSet.next()) {
                 getManufacturer(resultSet);
```

Comment:
I mean the following:
```suggestion
                manufacturer  = getManufacturer(resultSet);
```